text/stacks/0004-jammy-jellyfish.md

```diff
@@ -0,0 +1,106 @@
+# Stacks based on Ubuntu 2022.04: Jammy Jellyfish
+
+## Summary
+
+A set of stacks based on the Ubuntu 2022.04 LTS (Jammy Jellyfish) release base
+image should be developed, released, and maintained by the Stacks subteam. Like
+the existing Bionic stacks, these new stacks should come in "full", "base", and
+"tiny" variants with similar, if not identical, sets of packages pre-installed.
+
+## Motivation
+
+Ubuntu provides a long-term-support (LTS) release every 2 years in April. These
+releases are supported by Canonical for 5 years. The current Bionic base image,
+first released in April 2018, will therefore be going out of support at the end
+of March 2023. In order to ensure that Paketo continues to provide a supported
+Ubuntu-based stack for our users, we should produce a new set of stack images,
+based on Jammy, alongside the current Bionic stacks.
+
+## Detailed Explanation
+
+### Variants
+
+The Jammy stack will be delivered in 3 variants that align to those already
+offered in the Bionic stack. The variants will be called `base`, `full`, and
+`tiny` just as they are in the Bionic stack. Each should be developed to offer
+roughly the same set of OS-level package support as their Bionic equivalent.
+
+### Stack IDs
+
+Each of these stacks will declare their stack ID as `io.paketo.stack.jammy`.
+This choice differs from Bionic in that we will **NOT** differentiate between
+the `tiny` and `base/full` variants using the stack ID.
```

Comment:
How is this different? I think that base and full use `"io.buildpacks.stack.id": "io.buildpacks.stacks.bionic"`, and just tiny has a different ID. Are there any ramifications of changing that?

Reply:
Its different in that the "tiny" variant will have the same stack ID as "base" and "full".

As far as ramifications go, I think the experience for "tiny" now becomes like that for "base" and "full" in that there are some buildpacks that will operate successfully on "full", but not on "base", and that we document that in the builder or documentation for the buildpack itself. Having a separate stack ID just created extra metadata that needed to be added and maintained in all the buildpacks that supported "tiny", with very little technical benefit.